consensus/common_test.go

```diff
@@ -317,67 +317,138 @@ func ensureNoNewEvent(ch <-chan interface{}, timeout time.Duration,
 	}
 }
 
-func ensureNoNewStep(stepCh <-chan interface{}) {
+func ensureNoNewEventOnChannel(ch <-chan interface{}) {
+	ensureNoNewEvent(ch, ensureTimeout, "We should be stuck waiting, "+
+		"not receiving new event on the channel")
```

Comment:
This breaking is a bit clunky, it would be better to use this pattern:

``` go
	ensureNoNewEvent(
		ch,
		ensureTimeout,
		"We should be stuck waiting, not receiving new event on the channel",
	)
```